mlir/test/Dialect/EmitC/invalid_ops.mlir

```diff
@@ -387,3 +387,11 @@ func.func @logical_or_resulterror(%arg0: i32, %arg1: i32) {
   %0 = "emitc.logical_or"(%arg0, %arg1) : (i32, i32) -> i32
   return
 }
+
```

Comment:
Can you update the error message in `test_assign_to_non_variable` above and add a similar ` test_assign_to_array`.

Reply:
I updated the error message in `test_assign_to_non_variable`.
What is the change you'd like to see for
```
func.func @test_assign_to_array(%arg1: !emitc.array<4xi32>) {
  %v = "emitc.variable"() <{value = #emitc.opaque<"">}> : () -> !emitc.array<4xi32>
  // expected-error @+1 {{'emitc.assign' op cannot assign to array type}}
  emitc.assign %arg1 : !emitc.array<4xi32> to %v : !emitc.array<4xi32>
  return
}
```
?

Reply:
Oh sorry, I missed that one.